assets/dev/js/frontend/utils/anchors.js

```diff
@@ -84,7 +84,5 @@ module.exports = elementorModules.ViewModule.extend( {
 
 	onInit: function() {
 		elementorModules.ViewModule.prototype.onInit.apply( this, arguments );
-
-		this.bindEvents();
```

Comment:
This method is already triggered from ViewModule parent class.
It causes the events to trigger twice.

Reply:
Before I fixed the bug, the animate function in line 75 ran four times.